universite-du-quebec-a-montreal.csl

```diff
@@ -993,9 +1021,15 @@
       <else-if type="webpage" match="any">
         <text variable="volume" prefix=" (" suffix=")"/>
       </else-if>
+      <else-if type="book" match="all" variable="original-date">
+        <group delimiter=" " prefix=" (" suffix=").">
+          <text value="Oeuvre originale publiée en "/>
+          <date form="text" variable="original-date" prefix="&amp;#169;"/>
+        </group>
+      </else-if>
     </choose>
   </macro>
-  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" givenname-disambiguation-rule="primary-name" collapse="year">
+  <citation et-al-min="3" et-al-use-first="1" initialize-with="." disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" givenname-disambiguation-rule="primary-name" collapse="year">
```

Comment:
I'd remove the `initialize-with` here -- afaict you have this on all names (and where you don't, you can add it) and people don't expect this on `citation`, so this will lead to all sorts of confusion in editing the style.

Reply:
I was thinking about initials that appear in citations when two authors must be disambiguated. I will remove it.

Reply:
You can add that to the `name` element as well (if it isn't already). They won't show up with `form="short"` unless for disambiguation.